packages/browser-wallet/src/popup/pages/Account/AccountActions/AccountActions.tsx

```diff
@@ -50,34 +51,55 @@ function ActionLinks({ children }: ActionLinksProps) {
     );
 }
 
-interface Props {
-    className: string;
-    setDetailsExpanded: React.Dispatch<React.SetStateAction<boolean>>;
+type ActionProps = {
+    children: ReactNode;
+    to: string;
+    title: string;
+    disabled?: boolean;
+    end?: boolean;
+    onClick?: () => void;
+};
+
+function Action({ children, to, title, disabled = false, end = false, onClick = undefined }: ActionProps) {
```

Comment:
I think you always provide the `disabled` variable, so you don't need to provide a default value.